x-pack/plugins/spaces/public/constants.ts

```diff
@@ -16,10 +16,20 @@ export const getSpacesFeatureDescription = () => {
         'Organize your dashboards and other saved objects into meaningful categories.',
     });
   }
-
   return spacesFeatureDescription;
 };
 
+let spacesLoadingMessage: string;
+
+export const getSpacesLoadingMessage = () => {
+  if (!spacesLoadingMessage) {
+    spacesLoadingMessage = i18n.translate('xpack.spaces.loadingMessage', {
+      defaultMessage: 'Loading...',
+    });
+  }
+  return spacesLoadingMessage;
+};
+
```

Comment:
What's the reasoning behind this getter method that always seems to return a constant value?

Reply:
I just used the same style that was here for the placeholder description text. The implementation allows the text to be referenced from multiple places without causing multiple i18n lookups. I suppose my thinking was to keep the convention in that component, though I see no explicit reason for it.

Reply:
I removed the function from constants.ts and used a more conventional implementation local to usage (in spaces_description.tsx)